tiledb/sm/serialization/query.cc

```diff
@@ -531,20 +571,18 @@ Status subarray_partitioner_from_capnp(
   auto sr_reader = state_reader.getSingleRange();
   const unsigned num_sr = sr_reader.size();
   for (unsigned i = 0; i < num_sr; i++) {
-    auto subarray_reader_ = sr_reader[i];
-    state->single_range_.emplace_back(
-        array, layout, query_stats, dummy_logger, true);
-    Subarray& subarray_ = state->single_range_.back();
-    RETURN_NOT_OK(subarray_from_capnp(subarray_reader_, &subarray_));
+    auto subarray_reader = sr_reader[i];
```

Comment:
`subarray_reader` is already declared. This may generate a warning on some future compiler. Better just to give it it's own name. Admittedly `subarray_reader_` was bad, since it looks like a member variable.